tests/gateway-tests/test/apitests/gateway_events_test.go

```diff
@@ -66,4 +77,34 @@ func TestGatewayEvents(t *testing.T) {
 		assert.Equal(t, "31109198-4d69-4ae0-972d-76117f3748c8", publishResponse.EventId)
 	})
 
+	t.Run("should get all subscribed events", func(t *testing.T) {
+		//given
+		client, e := testkit.NewSubscriptionsClient()
+		require.NoError(t, e)
+
+		err := client.Create(config.Namespace, config.Application, eventType)
+		require.NoError(t, err)
+
+		url := config.EventServiceUrl + "/" + config.Application + "/v1/events/subscribed"
+
+		req, err := http.NewRequest(http.MethodGet, url, nil)
+		require.NoError(t, err)
+
+		//when
+		response, err := http.DefaultClient.Do(req)
+		require.NoError(t, err)
+
+		//then
+		var events SubscribedEvents
+
+		err = json.NewDecoder(response.Body).Decode(&events)
+		require.NoError(t, err)
+
+		assert.Equal(t, http.StatusOK, response.StatusCode)
+		assert.Equal(t, eventType, events.EventsInfo[0].Name)
```

Comment:
I do not think you should assume that there will be only one subscription while running the test.